templates/default/070-components/UI-framework/MainControls/_ui-component_mode_info.scss

```diff
@@ -31,31 +38,37 @@ $il-mode-info-shadow: 0px 2px 2px 0px rgba(128, 128, 128, 1);
   }
 }
 
-
-.il-mode-info span.il-mode-info-content {
-  margin: auto;
+.il-mode-info-content,
+.il-mode-info-close {
   background-color: $il-mode-info-color;
-  padding: $il-padding-base-vertical $il-padding-base-horizontal $il-padding-base-vertical $il-padding-base-horizontal;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: $il-mode-info-height;
```

Comment:
Maybe use min-height so multi-line text can push the area down? Unfortunately, this wouldn't push down the part of the bar with the close button in it. How likely is multi-line text in the mode info (especially on mobile)?